### ISSUE TITLE
Improve workshop accessibility

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#558]
At MakerDay on Saturday one of the attendees had a wheelchair, which wouldn't get past the tables in the existing format.  (It also wouldn't get through the kitchen doorway, but that's trickier to fix...)

The workshop is now in the "most quickly implemented wheelchair accessible layout" option, which involves pushing the island of tables against the woodwork bench side of the room, to free up room round the other side.  (Thanks @stevesparrow and @ajlennon for the removal work)

This is something we'll be addressing during the move, but it'd be good to work out a temporary solution before then.  We could...

 * Leave the workshop as-is, which means there's no access to the woodwork bench and restricted access to the multimeter/sawing/etc. drawers and vinyl-cutter.  Some of the stuff from under the tables has been decanted into Dinky temporarily, so we'd need to find a more permanent home for it with this option
 * Revert to the previous layout, and clear more of the stuff stored(/hidden) under the tables to make it easier to swap the layouts round when needed
 * Find a better way of fitting in the tables in the middle that reduces the amount of space they take up
 * Do something else...

**Comment by stevesparrow** [comment]
It would be good to try the option with the tables facing inward with the
extending pieces facing inwards.  It might not be enough, but would be good
to try.  Happy to try this one quiet day.

I do really like the extra space without the middle expanse of table,
especially filled above and beneath.

**Comment by ajlennon** [comment]
I liked the layout to begin with, but was unhappy toward the end of the day.

You seemed to have done an excellent job of trapping my child in the space between the tables, and he seemed very happy with this. However he somehow managed to escape. Please try harder!

**Comment by huffeec** [comment]
I know it was appreciated! I'm sure we'll see the attendee again, they were drawing doughnut narwhal t-shirt designs on Sunday ready for the next visit.
If we can manage with limited access to the woodworking bench for now it does make the space more navigable for everyone. There is a pinch point near the knitic but apart from that it worked well.

**Comment by Robotorium** [comment]
Aside from the knitic pinch point, it worked surprisingly well for me (got brews made for me instead of having to make them myself, bonus) considering the sheer amount of stuff vs the size of the space. I do have some thoughts regarding the height of things (mostly "bugger, that's high"), but in terms of the desks could the protruding corners not fit together somehow to make the overall surface take up less space?

I was really chuffed though and as huffeec said I'm definitely coming back. I really appreciate the work that went into it as it stands, and while it isn't perfect I genuinely felt completely supported, which is more important to me than whether or not the space is ideal, if that makes sense.

**Comment by DoESsean** [comment]
This is going to be important moving into the new building.

It may be that the best option is to canvass our wheelchair users and other users who may have mobility issues and ask them to help us with the layout of the space.

**Comment by cheapjack** [comment]
Happy to help with this if I can after xmas

**Comment by Robotorium** [comment]
I discussed this with Bob at some length a couple of months ago

Op 19 dec. 2017 13:40 schreef "Ross Dalziel @cheapjack" <
notifications@github.com>:

**Comment by amcewen** [comment]
It feels like we're mostly getting excellent advice on specific issues elsewhere from @Robotorium, so I'm wondering if this issue still makes sense?

Are there thing we should be paying attention to when we're laying out furniture or the workshop?  If not, then I'm thinking we should probably close this issue.

**Comment by Robotorium** [comment]
Mostly leaving enough space between things, I think, and maybe making sure
piles of stuff don't block the path in a way that's easy to step around but
less easy when using wheels for feet.

Also somewhere on a flooring thread someone mentioned a height difference,
but I wasn't clear if it was a difference that would cause accessibility
issues.

Op zo 25 mrt. 2018 00:26 schreef Adrian McEwen <notifications@github.com>:

**Comment by johnmckerrell** [comment]
If we use standard new floorboards I think they’d be 18mm compared to the 25mm of the old one so there would be a 7mm step. That feels reasonable but shout if not as I believe there’s ways around it. 

--

**Comment by Robotorium** [comment]
Oh yeah, that's not an issue at all.

Op zo 25 mrt. 2018 08:47 schreef John McKerrell <notifications@github.com>:

> If we use standard new floorboards I think they’d be 18mm compared to the
> 25mm of the old one so there would be a 7mm step. That feels reasonable but
> shout if not as I believe there’s ways around it.
>
> --
> Sent from my mobile phone hence brevity and errors
>

**Comment by amcewen** [comment]
I think (/hope) we're getting better at thinking about this sort of stuff, so more specific issues (such as #1046 for a recent example) are probably the best way to address future improvements.

(And I reckon we should spawn new issues even for things like "area X is too cluttered and not accessible" - if the clutter can't be readily and immediately moved out of the way - as that's no different to "the bandsaw needs a new blade" or whatever...)
